src/schema/firefox-schemas.js

```diff
@@ -0,0 +1,189 @@
+const FLAG_PATTERN_REGEX = /^\(\?[im]*\)(.*)/;
+
+// Reference some functions on inner so they can be stubbed in tests.
+export const inner = {};
+
+function stripFlagsFromPattern(value) {
+  // TODO: Fix these patterns and remove this code.
```

Comment:
I added a comment about this to #1130, we'll want to replace them there until they get updated in Firefox.